### ISSUE TITLE
Change cell on frozed columns with `setCell()` not working

### ISSUE THREAD

**Issue by PaolaRuby** [tonytomov/jqGrid#1002]
After using `setCell()` on frozen column, the only way to update value is with 'reloadGrid',
is this an expected behavior?

Example: https://jsfiddle.net/4efusm3h/

**Comment by tonytomov** [comment]
Yes, currently  this is "expected" behavior. This apply not only on saveCell, but ont setRowData, addRowData and delRowData methods. 

Hope we will fix this ASAP.

**Comment by tonytomov** [comment]
Hello,

We have fixed the problem in setCell - https://github.com/tonytomov/jqGrid/commit/08682495c3d47fbd5c372eacc5b336128a92e7c1 . You can test it and let us know if it is working for you.

Thank you
Best Regards

**Comment by PaolaRuby** [comment]
@tonytomov sorry, the same behavior persists

https://jsfiddle.net/zup5t6kg/1/

**Comment by tonytomov** [comment]
Hello,

Thank you. Sorry it was a "last minute" error. It should work now.

Best Regards,
T. Tomov

**Comment by tonytomov** [comment]
Hello Paola,
It is really wired.I hope this time it is ok. 

Thank you

Best Regards
Tony

**Comment by PaolaRuby** [comment]
It seems that it is already working, thanks
